packages/facility-language-server/package.yaml

```
@@ -0,0 +1,34 @@
+---
+name: facility-language-server
+description: |
+  Facility Service Definition language. This version of Facility Service Definition
+  requires dotnet (.NET 6.0) to be installed.
+homepage: https://facilityapi.github.io/
+licenses:
+  - MIT
+languages:
+  - Facility Service Definition
+categories:
+  - LSP
+
+source:
+  id: pkg:github/FacilityApi/FacilityLanguageServer@v2.1.0
+  asset:
+    - target: darwin_x64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: darwin_arm64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: linux_x64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: linux_arm64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: win_x64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: win_arm64
+      file: Facility.LanguageServer.zip:libexec/
```

Apologies for the back and forth on this one, but we can simplify this to a single platform-agnostic expression:

```suggestion
  asset:
    file: Facility.LanguageServer.zip:libexec/
```